ttnn/cpp/ttnn/operations/core/work_split/work_split_tilize.hpp

```diff
@@ -154,7 +154,7 @@ struct FullRep {
 
     FullRep(uint32_t n_rows, uint32_t n_pads, uint32_t times, uint32_t pads_mul, uint32_t times_total) :
         rep{n_rows / 32, n_rows % 32, n_pads / 32, times},
-        pad{0, 0, (n_rows + n_pads) * pads_mul, 1},
+        pad{0, 0, (n_rows + n_pads) * pads_mul / 32, 1},
```

Comment:
Is this 32 because of TILE shape. If so can we use the tensor tile shape api to get the tile height instead of hard coded number. We will support other tile shapes soon so good to use the api.